### DISCUSSION TITLE
MacOS Version

### DISCUSSION THREAD

**Discussion by cukabeka** [d2phap/ImageGlass#719] [General] [answered]
Hi,

I look for an alternative to picasa and found ImageGlass which looks promising! Thanks folks!

Is there a chance you will compile something for MacOS one day? I saw that there seems to be a chance to have this happen if this is a .NET app, so maybe via https://www.mono-project.com/ ? 
And I found https://github.com/d2phap/ImageGlass/issues/439 which looks as if there was already some thought in this direction? 

Regards!

**Comment by d2phap** [comment, marked as answer]
hi @cukabeka 
It's too big to port the current project to other framework/platform. I am still looking for some possible solutions, but no plan for the near future yet.

**Reply by eugenesvk** [reply]
Does rewriting the app from scratch and moving to .NET 6.0 in the new version 9 of ImageGlass change anything in this regard? Would it be possible to write a cross-platform app or is it still too much work and not planned?
Thanks!

**Reply by d2phap** [reply]
The v9 still uses WinForms framework which is Windows-only. I have a plan to use .NET MAUI in the future version (v10 or v11) and drop suport for Windows 7, 8, 8.1. .NET MAUI is still in early stage, hopefully it will get better and easier to develop cross-platform apps.

**Reply by cukabeka** [reply]
Great, thanks for looking into this and the update. Looking forward to a future cross-platform version.
